config/jobs/image-pushing/k8s-staging-sig-storage.sh

```diff
@@ -38,7 +38,7 @@ readonly BROKEN_REPOS=(
     kubernetes-csi/csi-driver-iscsi
     kubernetes-csi/csi-driver-nfs
     kubernetes-csi/csi-proxy
-    kubernetes-sigs/container-object-storage-interface-api
+    kubernetes-sigs/container-object-storage-interface-controller
```

Comment:
Why removing kubernetes-sigs/container-object-storage-interface-api?

Reply:
I do not need post submit job for api as I will not be pushing any images with that repo.

Reply:
Ok